SIL.Windows.Forms/SIL.Windows.Forms-Designer.csproj

```diff
@@ -72,9 +72,6 @@
       <HintPath>..\packages\L10NSharp.4.0.0\lib\net461\L10NSharp.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <Reference Include="NAudio">
-      <HintPath>..\packages\NAudio.1.8.4\lib\net35\NAudio.dll</HintPath>
-    </Reference>
```

Comment:
I see you removed references to NAudio in a few places. Was this just not being used at all? (I think that's the case because I believe the audio stuff on Windows is just using irrKlang.) if there is any place in the common libraries where we're using NAudio, we should evaluate that. I created a fork from the official NAudio because I fixed a bug, and every time they "accepted" my patch, they didn't include the unit tests and they changed the code in ways that didn't actually fix the bug.

Reply:
It wasn't used anywhere in SIL.Windows.Forms - I guess it was a left-over. NAudio is used and referenced in SIL.Media.